lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -525,6 +525,14 @@ ThreadContext::~ThreadContext()
 #ifdef ENABLE_SCRIPT_DEBUGGING
         Assert(this->debugManager == nullptr);
 #endif
+
+#if ENABLE_CONCURRENT_GC
+        //OS Bug# 15975576 [Not Exploitable] JSHost Fuzzing: UM_WRITEAV_UNKNOWN_EXPLOITABLE_chakratest.dll!Memory::HeapBlockMap32::Mark_0,0_
+        // We have multiple hits in the JShost Fuzzing but we believe this is only an issue for JSHOST test harness. Adding assert to verify that we never hit this in crawler.
```

Comment:
Is this comment temporary? will it go away once we identify the issue? AssertMsg is self explanatory to have it here.

Reply:
I think we will keep the asserts here for a while. I will remove the comments. 

---
In reply to: [194576596](https://github.com/Microsoft/ChakraCore/pull/5296#discussion_r194576596) [](ancestors = 194576596)